writer.go

```diff
@@ -145,16 +161,18 @@ type WriterStats struct {
 	BatchSize  SummaryStats  `metric:"kafka.writer.batch.size"`
 	BatchBytes SummaryStats  `metric:"kafka.writer.batch.bytes"`
 
-	MaxAttempts       int64         `metric:"kafka.writer.attempts.max"       type:"gauge"`
-	MaxBatchSize      int64         `metric:"kafka.writer.batch.max"          type:"gauge"`
-	BatchTimeout      time.Duration `metric:"kafka.writer.batch.timeout"      type:"gauge"`
-	ReadTimeout       time.Duration `metric:"kafka.writer.read.timeout"       type:"gauge"`
-	WriteTimeout      time.Duration `metric:"kafka.writer.write.timeout"      type:"gauge"`
-	RebalanceInterval time.Duration `metric:"kafka.writer.rebalance.interval" type:"gauge"`
-	RequiredAcks      int64         `metric:"kafka.writer.acks.required"      type:"gauge"`
-	Async             bool          `metric:"kafka.writer.async"              type:"gauge"`
-	QueueLength       int64         `metric:"kafka.writer.queue.length"       type:"gauge"`
-	QueueCapacity     int64         `metric:"kafka.writer.queue.capacity"     type:"gauge"`
+	MaxAttempts          int64         `metric:"kafka.writer.attempts.max"       		type:"gauge"`
+	Retry                int64         `metric:"kafka.writer.retries.max"        		type:"gauge"`
+	RetryBackoffInterval time.Duration `metric:"kafka.writer.retrybackoff.interval"    	type:"gauge"`
```

Comment:
Some naming inconsistencies here with the existing attributes.  `MaxAttempts` and `RequiredAcks` are plural but `Retry` is singular but the `metric` name is plural?  I would recommend renaming to `Retries` for consistency.

Reply:
It's because there already is a stat called `Retries`, that's watching the count of `Retries`. 

But I agree that the naming is off. I might name it something like `MaxRetries`, but that feels misleading (makes it similar to `MaxAttempts` tho).